v2.1/export.md

```diff
@@ -0,0 +1,160 @@
+---
+title: EXPORT
+summary: Export tabular data from CockroachDB cluster in CSV format.
+toc: false
+---
+
+The `EXPORT` [statement](sql-statements.html) exports tabular data to CSV files.
+
+{{site.data.alerts.callout_danger}}The <code>EXPORT</code> feature is only available to <a href="https://www.cockroachlabs.com/product/cockroachdb/">enterprise</a> users. Also note that this feature in currently under development and is slated for full release in CockroachDB 2.1. Hence the feature flags and behavior are subject to change. {{site.data.alerts.end}}
```

Comment:
`this feature in` > `this feature is`

Reply:
Also, a nit: `Hence` feels unconventional for our docs. I'd just say:
> The current feature flags and behavior are subject to change.

Reply:
Done.